cmake/summary.cmake

```diff
@@ -21,7 +21,6 @@ function (onnx_print_configuration_summary)
   message(STATUS "  ONNX NAMESPACE                    : ${ONNX_NAMESPACE}")
   message(STATUS "  ONNX_USE_LITE_PROTO               : ${ONNX_USE_LITE_PROTO}")
   message(STATUS "  USE_PROTOBUF_SHARED_LIBS          : ${ONNX_USE_PROTOBUF_SHARED_LIBS}")
-  message(STATUS "  Protobuf_USE_STATIC_LIBS          : ${Protobuf_USE_STATIC_LIBS}")
```

Comment:
Why remove this line?

Reply:
Now Protobuf_USE_STATIC_LIBS is set by ONNX_USE_PROTOBUF_SHARED_LIBS  and has the opposite value of ONNX_USE_PROTOBUF_SHARED_LIBS. It should not be set directly by the user, so it's not printed.